client/gutenberg/extensions/related-posts/edit.jsx

```diff
@@ -14,7 +22,52 @@ import { withSelect } from '@wordpress/data';
  * Internal dependencies
  */
 import { __ } from 'gutenberg/extensions/presets/jetpack/utils/i18n';
-import { DEFAULT_POSTS, MAX_POSTS_TO_SHOW } from './constants';
+
+export const MAX_POSTS_TO_SHOW = 3;
+
+class PlaceholderPostEdit extends Component {
+	render() {
+		const { displayDate, displayContext, displayThumbnails } = this.props;
+		const previewClassName = 'related-posts__preview';
+
+		return (
+			<div className={ `${ previewClassName }-post` }>
+				{ displayThumbnails && (
+					<Button className={ `${ previewClassName }-post-image-placeholder` } isLink>
+						<span
+							className={ `${ previewClassName }-post-image-placeholder-icon` }
+							aria-label={ __( 'Placeholder image' ) }
+						>
+							<SVG
+								xmlns="http://www.w3.org/2000/svg"
+								width="24"
+								height="24"
+								viewBox="0 0 24 24"
+								focusable="true"
```

Comment:
I'm curious why `focusable="true"` here? Isn't the parent `Button` already focusable?

Reply:
This may have been an oversight, as I was planning to use a `title` tag for the SVG but ended up using `aria-label` for the `span` instead.

The `focusable` attribute also doesn't do anything with the `Svg` Gutenberg component.

Reply:
🍿 just chiming in to say that I love watching @aldavigdis talk about accessibility because I almost always learn lots of valuable stuff.

what's the effective difference between using `title` on the SVG vs. `aria-label` on the `span`?